### PR TITLE
Correct PDF link

### PR DESCRIPTION
**Pull request by sguitardude**
Update the PDF link to be the same as the link for the image.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/technical-reference/technical-diagrams.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This poster shows the recommended MinRole topologies in a SharePoint on-premises
...
+|[![Click to view and download this poster about SharePoint Server MinRole topologies.](../media/9f753f44-a084-4a98-9ed4-e233d5cdf9e8.png)          ](http://download.microsoft.com/download/5/F/4/5F4107F5-ECFA-49EA-A02A-799D9EAFB03C/MinRole_Topology_Tabloid_Nov2018.pdf) <br/> ![PDF file](../media/ITPro_Other_PDFicon.png)[PDF](http://download.microsoft.com/download/5/F/4/5F4107F5-ECFA-49EA-A02A-799D9EAFB03C/MinRole_Topology_Tabloid_Nov2018.pdf) | ![Visio file](../media/ITPro_Other_VisioIcon.jpg)[Visio](https://go.microsoft.com/fwlink/?linkid=837452) <br/> |This poster shows the different recommended MinRole topologies that can be deployed in a SharePoint Servers 2016 and 2019 environments. It also shows the associated services that are provisioned with each role type.  <br/> |
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/technical-reference/technical-diagrams.md]:
```suggestion
|[![Click to view and download this poster about SharePoint Server MinRole topologies.](../media/9f753f44-a084-4a98-9ed4-e233d5cdf9e8.png)](http://download.microsoft.com/download/5/F/4/5F4107F5-ECFA-49EA-A02A-799D9EAFB03C/MinRole_Topology_Tabloid_Nov2018.pdf) <br/> ![PDF file](../media/ITPro_Other_PDFicon.png)[PDF](http://download.microsoft.com/download/5/F/4/5F4107F5-ECFA-49EA-A02A-799D9EAFB03C/MinRole_Topology_Tabloid_Nov2018.pdf) | ![Visio file](../media/ITPro_Other_VisioIcon.jpg)[Visio](https://go.microsoft.com/fwlink/?linkid=837452) <br/> |This poster shows the different recommended MinRole topologies that can be deployed in SharePoint Servers 2016 and 2019 environments. It also shows the associated services that are provisioned with each role type.  <br/> |
```

Comment (nenonix) [inline SharePoint/SharePointServer/technical-reference/technical-diagrams.md]:
```suggestion
|[![Click to view and download this poster about SharePoint Server MinRole topologies.](../media/9f753f44-a084-4a98-9ed4-e233d5cdf9e8.png)          ](http://download.microsoft.com/download/5/F/4/5F4107F5-ECFA-49EA-A02A-799D9EAFB03C/MinRole_Topology_Tabloid_Nov2018.pdf) <br/> ![PDF file](../media/ITPro_Other_PDFicon.png)[PDF](http://download.microsoft.com/download/5/F/4/5F4107F5-ECFA-49EA-A02A-799D9EAFB03C/MinRole_Topology_Tabloid_Nov2018.pdf) | ![Visio file](../media/ITPro_Other_VisioIcon.jpg)[Visio](https://go.microsoft.com/fwlink/?linkid=837452) <br/> |This poster shows the different recommended MinRole topologies that can be deployed in SharePoint Server 2016 and 2019 environments. It also shows the associated services that are provisioned with each role type.  <br/> |
```

Comment (sguitardude) [reply SharePoint/SharePointServer/technical-reference/technical-diagrams.md]: Still learning this github system.  Did I incorrectly commit JohnFreelancer9's changes?  I don't see the option to also add nenonix7's changes.

**Pull request merged**